mmv1/products/dlp/api.yaml

```diff
@@ -1328,4 +1332,273 @@ objects:
+                                  properties:
+                                    - !ruby/object:Api::Type::Integer
+                                      name: year
+                                      description: Year of the date. Must be from 1 to 9999, or 0 to specify a date without a year.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: month
+                                      description: Month of a year. Must be from 1 to 12, or 0 to specify a year without a month and day.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: day
+                                      description: Day of a month. Must be from 1 to 31 and valid for the year and month, or 0 to specify a year by itself or a year and month where the day isn't significant.
+                                - !ruby/object:Api::Type::Enum
+                                  name: dayOfWeekValue
+                                  description: Represents a day of the week.
+                                  values:
+                                    - :MONDAY
+                                    - :TUESDAY
+                                    - :WEDNESDAY
+                                    - :THURSDAY
+                                    - :FRIDAY
+                                    - :SATURDAY
+                                    - :SUNDAY
+                        - !ruby/object:Api::Type::NestedObject
+                          name: redactConfig
+                          description: |
+                            Redact a given value. For example, if used with an InfoTypeTransformation transforming PHONE_NUMBER, and input 'My phone number is 206-555-0123', the output would be 'My phone number is '.
+                          properties: [] # Meant to be an empty object with no properties - see here : https://cloud.google.com/dlp/docs/reference/rest/v2/projects.deidentifyTemplates#redactconfig
+                          # The fields below are necessary to include the "redactConfig" transformation in the payload
+                          # A side-effect is null values when the field is unused, see: https://github.com/hashicorp/terraform-provider-google/issues/13201
+                          send_empty_value: true
+                          allow_empty_object: true
+                        - !ruby/object:Api::Type::NestedObject
+                          name: characterMaskConfig
+                          description: |
+                            Partially mask a string by replacing a given number of characters with a fixed character. Masking can start from the beginning or end of the string. This can be used on data of any type (numbers, longs, and so on) and when de-identifying structured data we'll attempt to preserve the original data's type. (This allows you to take a long like 123 and modify it to a string like **3.
```

Comment:
```suggestion
                            Partially mask a string by replacing a given number of characters with a fixed character. Masking can start from the beginning or end of the string. This can be used on data of any type (numbers, longs, and so on) and when de-identifying structured data we'll attempt to preserve the original data's type. (This allows you to take a long like 123 and modify it to a string like **3).
```